modules/acquia_cms_tour/src/Form/InstallationWizardForm.php

```diff
@@ -366,7 +366,9 @@ protected function getFieldValueFromStorage(string $field_name, $empty_value = N
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $formController = $this->getCurrentFormController()['class'];
-    $this->classResolver->getInstanceFromDefinition($formController)->submitForm($form, $form_state);
+    if ($formController != "Drupal\acquia_cms_tour\Plugin\AcquiaCmsTour\GoogleAnalyticsForm") {
```

Comment:
@rajeshreeputra - We should not check for specific module here, It could any module so the logic should regardless of any modules config.

Reply:
Updated as per discussion